methods/eosio.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eosio",
-  "status": "registered",
+  "status": "deprecated",
```

Comment:
```suggestion
  "status": "registered",
```

Reply:
@Gimly-Caspar You are the currently listed change controller, if we don't hear from you, I will merge this change, and assume you have not authorized deprecation of eosio method.